app/views/wikis/edit.html.haml

```diff
@@ -9,5 +9,12 @@
       .form-group
         = f.label :body
         = f.text_area :body, class: "form-control"
+
+      - if current_user.role == 'admin' || current_user.role == 'premium'
```

Comment:
Consider using `.admin?` and `.premium?` methods instead of hard-coding the role name strings.